cmake/OpenVDBHoudiniSetup.cmake

```diff
@@ -335,12 +335,31 @@ if(Houdini_VERSION_MAJOR_MINOR VERSION_EQUAL 18.0)
   set(OPENVDB_HOUDINI_ABI 6)
 elseif(Houdini_VERSION_MAJOR_MINOR VERSION_EQUAL 18.5)
   set(OPENVDB_HOUDINI_ABI 7)
-elseif(Houdini_VERSION_MAJOR_MINOR VERSION_EQUAL 19.0)
-  set(OPENVDB_HOUDINI_ABI 8)
 else()
-  message(WARNING "Unknown version of Houdini, assuming OpenVDB ABI=8, "
-    "but if this not correct, the CMake flag -DOPENVDB_HOUDINI_ABI=<N> can override this value.")
-  set(OPENVDB_HOUDINI_ABI 8)
+  find_file(_houdini_openvdb_version_file "openvdb/version.h"
+    PATHS ${HOUDINI_INCLUDE_DIR}
+    NO_DEFAULT_PATH
+    NO_PACKAGE_ROOT_PATH
+    NO_CMAKE_PATH
+    NO_CMAKE_ENVIRONMENT_PATH
+    NO_SYSTEM_ENVIRONMENT_PATH
+    NO_CMAKE_SYSTEM_PATH)
```

Comment:
I think the only option you need to provide here is `NO_DEFAULT_PATH`

Reply:
Yes, you are correct. I completely missed that in the docs
> If NO_DEFAULT_PATH is specified, then no additional paths are added to the search.

---https://cmake.org/cmake/help/latest/command/find_file.html

I'll submit a fix for this shortly.

Reply:
Fixed in c81aa6e.